File: 백준/Bronze/2562. 최댓값/최댓값.py
import sys
input = sys.stdin.readline

arr = [int(input().rstrip()) for _ in range(9)]

max_value = arr[0]
max_idx = 0

for i in range(1, 9):
    if max_value < arr[i]:
        max_value = arr[i]
        max_idx = i

print(max_value)
print(max_idx + 1)

File: 백준/Bronze/2562. 최댓값/최댓값.cc
#include <stdio.h>

int main(void) {
    int arr[9] = {0,};
    for (int i = 0; i < 9; i++) scanf("%d", &arr[i]);

    int max_value = arr[0];
    int max_idx = 0;

    for (int i = 1; i < 9; i++) {
        if (max_value < arr[i]) {
            max_value = arr[i];
            max_idx = i;
        }
    }

    printf("%d\n%d", max_value, max_idx + 1);
    return 0;
}